### PR TITLE
Content media item filesize

### PR DESCRIPTION
**Pull request by nilllzz**

### CODE REVIEW

File: src/_common/content/components/media-upload/media-upload.ts [hunk 1]
```diff
@@ -111,6 +106,43 @@ export default class AppContentMediaUpload extends Vue {
 		);
 		if ($payload.success && $payload.mediaItems && $payload.mediaItems.length === 1) {
 			return new MediaItem($payload.mediaItems[0]);
+		} else if ($payload.success === false && $payload.errors.file) {
```
Comment (YLivay) [inline src/_common/content/components/media-upload/media-upload.ts:109]: `!$payload.success`, I'm not sure if we consistently return false or maybe just not return success at all in some cases.

File: src/_common/content/components/media-upload/media-upload.ts [hunk 2]
```diff
@@ -111,6 +106,43 @@ export default class AppContentMediaUpload extends Vue {
...
+				}
+			);
+
+			const maxWidth = sizePayload.maxWidth;
+			const maxHeight = sizePayload.maxWidth;
+
+			Growls.error({
+				title: this.$gettext('Oh no!'),
+				message: this.$gettextInterpolate(
+					"It looks like your image's filesize or dimensions are too large. The maximum allowed size is %{ width }x%{ height } pixels.",
```
Comment (YLivay) [inline src/_common/content/components/media-upload/media-upload.ts:127]: lets also output the max filesize.

Comment (nilllzz) [reply src/_common/content/components/media-upload/media-upload.ts:127]: Yeah. I'll add the parameter, but @hworld has to figure out the message.

Comment (hworld) [reply src/_common/content/components/media-upload/media-upload.ts:128]: It looks like your image's filesize or dimensions are too large. Its filesize must be less than %{filesize} and its dimensions less than NxN.

**Pull request merged**